app/timetables/models.py

```diff
@@ -175,3 +175,32 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Event(TimestampMixin):
+    """
+    Model representing event
+
+    Event represent a date or range of dates to which a
+    specific timetable will be active or functional.
+    """
+
+    name = models.CharField(max_length=150)
+    timetable = models.ForeignKey(Timetable, on_delete=models.CASCADE)
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
+
+    def clean(self):
+        if self.start_date >= self.end_date:
```

Comment:
@andela-snwuguru if you enter values that are not of DateTime in the admin, this part of the code would break as it would be trying to compare None type object. Let us check if self.start_date and self.end_date are not NoneType before doing this comparison. A scenario is on this issue https://github.com/teamtaverna/core/issues/62